example/app/api/serializers.py

```diff
@@ -0,0 +1,28 @@
+from django.db.models import Field, FileField, ImageField
+from rest_framework import serializers
+
+import saritasa_s3_tools.django
+
+from .. import models
+
+
+class ModelWithFilesSerializer(serializers.ModelSerializer):
+    """Serializer to show info model with files."""
+
+    class Meta:
+        model = models.ModelWithFiles
+        fields = "__all__"
+
+    @property
```

Comment:
It seem confusing that specific serializer should update this mapping, let's move this logic to lib itself